editions/tw5.com/tiddlers/filters/lookup Operator.tid

```diff
@@ -1,24 +1,26 @@
 caption: lookup
 created: 20170907103639431
-modified: 20170907144703051
+modified: 20210116081305739
 op-input: a [[selection of titles|Title Selection]]
-op-output: the lookup values corresponding to each input title
-op-parameter: prefix applied to input titles to yield title of lookup tiddler from which value is retrieved
-op-parameter-name: P
-op-purpose: applies a prefix to each input title to yield the title of a tiddler from which the final value is retrieved
-op-suffix: the default value to be used for missing lookups
-op-suffix-name: D
+op-output: the lookup values corresponding to each lookup tiddler
+op-parameter: prefix applied to input titles to yield title of lookup tiddler from which value is retrieved. Now accepts 1 or 2 parameters, see below for details
+op-parameter-name: P, T
+op-purpose: applies a prefix to each input title to yield the title of a tiddler from which the final value is retrieved. With a single parameter, the default field is "text" and the default index is "0". If a second parameter is provided, that becomes the target field or index.
+op-suffix: the default value to be used for missing lookups. This operator can now accept a second suffix, see below for details
+op-suffix-name: D, I
 tags: [[Filter Operators]]
 title: lookup Operator
 type: text/vnd.tiddlywiki
 
-<<.from-version "5.1.15">>
+<<.from-version "5.1.24">>
```

Comment:
Hmmm, I'm not sure, if the "old" behaviour should be overwritten. I think there needs to be "from 5.1.15" AND a "from 5.1.24" 

Since TW 5.1.23 uses the description 5.1.15 ..

Reply:
You may move "from 5.1.24" to the top and "from 5.1.15" to the bottom of the tiddler

Reply:
Hi @joshuafontany I think @pmario is right, it would be helpful to retain the old version marker, if we can distinguish between the features to which they apply.